library/waku.nim

```diff
@@ -0,0 +1,399 @@
+
+import
+  std/[sequtils,times,strformat,json,options],
+  strutils,
+  os
+import
+  chronicles,
+  chronos,
+  libp2p/crypto/secp,
+  stew/shims/net,
+  eth/net/nat as todo_delete_this_module
```

Comment:
Leftover? :)

Reply:
I will clean the `setupNat`, once this is merged, in a separate PR :)